cmake/Platform/Core/BoardFlags/CompilerFlagsSetter.cmake

```diff
@@ -1,4 +1,19 @@
 # ToDo: Comment
+
```

Comment:
Could you please document it like all other functions?

Reply:
Yes, I can document it. 

The line
```
# ToDo: Comment
```
is left over from the also uncommented function `set_board_compiler_flags`.